index.html

```diff
@@ -207,5 +208,47 @@ <h4>Skills</h4>
 
   </div>
   </section>
+  
+        <!--Form section-->
+        <section class="form">
+          <div class="info">
+
+           <div class="form-text"><h3>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3></div>
+           <form action="https://formspree.io/f/xeqwozwl" method="post" >
+           
+             <input id="name" type="text" class="input"  name="user_name" placeholder="Full name" required maxlength="30">
+
+             <input id="email" type="email" class="input" name="user_email" placeholder="Email address" required>
+
+             <textarea id="message" class="msg" name="user_message" placeholder="Enter text here" required maxlength="500"></textarea>
+
+             <button type="submit" class="form-bttn" id="btnid">Get In Touch</button>
+             
```

Comment:
- [ ] Button interactions such as hover, active (when it's pressed), disabled and enable (default state) are styled as shown in the Figma template. Kindly work on it. :+1: 

![2023-02-21_01-55](https://user-images.githubusercontent.com/70483835/220323219-5b15a1df-cac5-4ebd-8c05-2fb3fce8ac58.png)